src/PerfView/CommandLineArgs.cs

```diff
@@ -368,6 +370,8 @@ private void SetupCommandLine(CommandLineParser parser)
                 "Useful for trimming large ETL files. 1M typically yields 300-400 Meg of data considered.");
             parser.DefineOptionalQualifier("SkipMSec", ref SkipMSec, "Skips the first N MSec of the trace.  " +
                 "Useful for trimming large ETL files in conjunction with the /MaxEventCount qualifier.");
+            parser.DefineOptionalQualifier("StartTime", ref StartTime, "First event timestamp.");
```

Comment:
Can we add a bit more detail to the usage information of StartTime and EndTime so that when these arguments show up in the help it's clear to folks what they are?  Also, since these are only supported in BPerf files today, we should clarify that - perhaps by saying something like "The start date and time used to filter events of the input trace for formats that support this."